03-specifying-sample-designs.Rmd

```diff
@@ -143,7 +145,7 @@ $$se(\hat{p})=\sqrt{\frac{\hat{p}(1-\hat{p})}{n}} $$
 
 #### The syntax {-} 
 
-If you had a sample that was drawn through SRSWR and had no nonresponse or other weighting adjustments, in R, you should specify this design as:
+If you had a sample that was drawn through SRSWR and had no nonresponse or other weighting adjustments, in R, you should specify this design as:  
```

Comment:
Should we call out any differences between SRS & SRSWR?  If there are any?  Is it just a lack of fpc?

Reply:
Yes do this.